.github/workflows/backport-trigger.yml

```diff
@@ -0,0 +1,23 @@
+name: Backport Trigger
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  launchBackportBuild:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '@vs-mobiletools-engineering-service2 backport')
```

Comment:
@cadsit can't we use a shorter handle ?

Reply:
@spouliot Yes -- you can replace `@vs-mobiletools-engineering-service2` on line 10 with any text you'd like to trigger a build

Reply:
@mandel-macaque my choices, in preference order, are:
* @mandel-macaque
* @manuel
* @manual
* @sudo

Reply:
+1 for @Manual

Reply:
Yep let's rename it! @vs-mobiletools-engineering-service2 is jus tooooooooo long, looks like `@sudo` is free heh

Reply:
@spouliot @dalexsoto shall we call it @manual-macaque? Now serious, if any text is ok, I wonder if we can do /azp and use that command for everything

Reply:
Maybe, @azp, since with the / it will trigger the other bot :(

Reply:
To me honest, Manuel is perfectly valid :)

![](https://media.giphy.com/media/MRu6tLtQHi3zq/giphy.gif)